docs/source/blocks/blocks-core/teaser.md

```diff
@@ -0,0 +1,152 @@
+---
+myst:
+  html_meta:
+    "description": "Volto Teaser Block description and developer notes"
+    "property=og:description": "Volto Teaser Block description and developer notes"
+    "property=og:title": "Volto Teaser Block"
+    "keywords": "Volto, React, blocks, teaser, Plone"
+---
+
+(teaser-block)=
+
+# Teaser Block
+
+Volto comes with a Teaser block that given a content object as target is able to pull information from it and show it in the block.
+By default, this information is the `title`, `description`, `preview_image` and shows them in a summary (Image on left, title, description) way.
```

Comment:
```suggestion
By default, this information includes the `title`, `description`, and `preview_image`.
It shows them in a summary layout, with the image on the left, and the title and description on the right, with the description below the title.
```

Please add a screenshot. For syntax https://6.docs.plone.org/contributing/myst-reference.html#images-and-figures Also screenshots need to be kept at <=760 pixels wide, if possible, so that they are legible in the docs.